Deprecate tmb-compare module (#1298)
AlexsLemonade/OpenPBTA-analysis: analyses/README.md

| [`survival-analysis`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/survival-analysis) | TBD | *In progress*. Provides survival analysis template and performs several different survival analyses | N/A | Yes |
| [`telomerase-activity-prediction`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/telomerase-activity-prediction)| `pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds` <br> `pbta-gene-expression-rsem-fpkm-collapsed.polya.rds	` <br> `pbta-gene-counts-rsem-expected_count.stranded.rds` <br> `pbta-gene-counts-rsem-expected_count.polya.rds` | Quantify telomerase activity across pediatric brain tumors (part of [#148](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/148)) | `results/TelomeraseScores_PTBAPolya_counts` <br> `results/TelomeraseScores_PTBAPolya_FPKM.txt` <br> `results/TelomeraseScores_PTBAStranded_counts.txt` <br> `results/TelomeraseScores_PTBAStranded_FPKM.txt` <br> `results/EXTENDScores_{broad_histology}.tsv` | Yes |

> | [`tmb-compare`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/tmb-compare) | `pbta-snv-consensus-mutation-tmb-coding.tsv` | *Deprecated.* Compares PBTA tumor mutation burden to adult TCGA data. | N/A | Yes (figure only) | 

Reviewer: I feel like this more accurately captures what's going on:
```suggestion
| [`tmb-compare`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/tmb-compare) | `pbta-snv-consensus-mutation-tmb-coding.tsv` | *Deprecated.* Compares PBTA tumor mutation burden to adult TCGA data. | N/A | Not directly, similar figure generated in `figures/` | 
```

Author: 👍
